Code/Tools/ProjectManager/Platform/Mac/PAL_mac.cmake

```diff
@@ -5,3 +5,16 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 #
 #
+
+add_executable(O3DE_SDK Platform/Mac/O3DE_SDK_Launcher.cpp)
+
+set_target_properties(O3DE_SDK PROPERTIES
+    XCODE_ATTRIBUTE_ENABLE_HARDENED_RUNTIME YES
+    XCODE_ATTRIBUTE_CODE_SIGN_INJECT_BASE_ENTITLEMENTS NO
+)
+
+ly_target_link_libraries(O3DE_SDK
+    PRIVATE
+        AZ::AzCore
+        AZ::AzFramework
+)
```

Comment:
the target should be declared with `ly_add_target` so we dont run into issues in some scenarios, e.g. `ly_add_target` adds the target to some list so we install it.

Reply:
Yeah. This is currently being added in a .cmake file. When I move it to Code/Tools/BundleLauncher, I can use ly_add_target()